src/wireviz/DataClasses.py

```diff
@@ -11,15 +11,30 @@
 class Image:
     # Attributes of the image object <img>:
     src: str
-    scale: Optional[str] = "false"  # false | true | width | height | both
+    scale: Optional[str] = None  # false | true | width | height | both
     # Attributes of the image cell <td> containing the image:
     width: Optional[int] = None
     height: Optional[int] = None
-    fixedsize: Optional[bool] = False
+    fixedsize: Optional[bool] = None
     # Contents of the cell <td> just below the image cell:
     caption: Optional[str] = None
     # See also HTML doc at https://graphviz.org/doc/info/shapes.html#html
 
+    def __post_init__(self):
+
+        if self.scale is None:
+            self.scale = "false" if self.width is None     and self.height is None \
+                else     "both"  if self.width is not None and self.height is not None \
+                else     "true" # When only one dimension is specified.
+
+        if self.fixedsize is None:
+            self.fixedsize = self.width is not None or self.height is not None
+
+        if self.width is None and self.height is not None:
```

Comment:
maybe `if self.width and not self.height:` is enough, and a bit more readable?
Since `width` and `height` are both `int`, only a value of 0 should yield false, in which case this would probably also be the appropriate behavior.
Same for line 35 of course.